### DISCUSSION TITLE
IS GPU support enabled for embeddings? Or to put in another way how do I enable GPU for embedding work?

### DISCUSSION THREAD

**Discussion by micos7** [mudler/LocalAI#1502] [Q&A]
I  use the bert model for embeddings. I build it  with`make BUILD_TYPE=cublas build`. When I test an embedding with a fairly large text it takes the same time as the CPU only build (default), Thanks.

**Comment by fakezeta** [comment]
Hi @micos7,

transformer backend supports CUDA acceleration for embeddings.
